### ISSUE TITLE
Transsiberian railway broken

### ISSUE THREAD

**Issue by DavidParkin** [lukasmonk/lucaschess#45]
While doing some tactics I looked at saving the puzzle via Options. No I can no longer get into tactics. Selecting 'Play' returns me to the main screen. Can I fix this without restarting? 

I now this was a coincidence. The supplied tactics file was incorrect.

**Comment by DavidParkin** [comment]
Any chance of a reply? Some idea of where this information is saved would be a start.

**Comment by lukasmonk** [comment]
From where specific menu option are you playing ?

**Comment by DavidParkin** [comment]
Train -> Long-term trainings -> Transsiberian Railway -> Play
Just returns to main screen. I assume some info is corrupted.
I am at From Kamyshlov to Talitsa
If you want more just tell me where to look.
Now to my Linux problem.

**Comment by DavidParkin** [comment]
Copied the Variables.pk into my Linux version got this
Traceback (most recent call last):
  File "./Code/QT/PantallaRoutes.py", line 156, in play
    self.procesador.playRoute(self.route)
  File "./Code/Procesador.py", line 854, in playRoute
    self.gestor.inicio(route)
  File "./Code/GestorRoutes.py", line 597, in inicio
    self.partida_objetivo = PGN.pgn_partida(tactica.fen, tactica.pgn)
  File "./Code/PGN.py", line 353, in pgn_partida
    partida = pgn.partida
AttributeError: UnPGN instance has no attribute 'partida'

**Comment by lukasmonk** [comment]
Thanks, at the end of August or early September I will be able to work on it.

**Comment by DavidParkin** [comment]
I looked a little further. There is a function in in PGNreader.py called read which processes move taken from, in my case, ./IntFiles/Routes/Transsiberian/level2. It is unable to cope with 0-0-0 for castling and requires O-O-O or o-o-o, which is the PGN standard. In the end I edited this file, it wasn't perfect but it allowed me to advance.
If it's your holiday, have a good one, and if I post again don't see it as pressure.
